packages/relay-service/lib/payloads.js

```diff
@@ -1,14 +1,34 @@
 import VerifiableCredential from '@docknetwork/sdk/verifiable-credential';
 import {cryptoWaitReady} from '@polkadot/util-crypto';
-import {getKeypairFromDoc} from '@docknetwork/universal-wallet/methods/keypairs';
+// import {getKeypairFromDoc} from '@docknetwork/universal-wallet/methods/keypairs';
```

Comment:
commented code

Reply:
I will push a separate PR fixing this commented code